### PR TITLE
[EG-1575] Configuration Obfuscator page updates for CE 4.4

### PR DESCRIPTION
**Pull request by ivanterziev-r3**
NB Needs to be propagated to CE 4.5 as part of the team rota work this week. The following aliases should be added to the front matter of the CE 4.5 page:

`- /docs/cenm/config-obfuscation-tool.html`
`- /docs/cenm/1.3/config-obfuscation-tool.html`

Also, some of the code examples in the CE 4.4 page are using CE 4.4.2 in the config file name, might be a good idea to change to 4.5 in the 4.5 page.

### CODE REVIEW

**Comment by ivanterziev-r3** [comment]
@JamesHR3 For better readability, it might be good to have a separate sub-section specifically discussing deobfuscation (even if it would repeat some info or just say "use the same commands/options as with obfuscation" etc.).

File: content/en/docs/corda-enterprise/4.4/tools-config-obfuscator.md [hunk 1]
```diff
@@ -56,108 +62,225 @@ config-obfuscator [-hiV] [--config-obfuscation-passphrase[=<cliPassphrase>]]
...
+  "keyStorePassword": "<{8I1E8FKrBxVkRpZGZKAxKg==:oQqmyYO+SZJhRkPB7laNyQ==}>",
   // (...)
 }
 ```
 
-These directives can be placed arbitrarily within string properties in the configuration file, with a maximum of one per line.
-For instance:
+### Deobfuscation based on configuration directives
+
+When run by a Corda node or service on a machine with the matching hardware address, the configuration is deobfuscated on the fly and interpreted as shown in the example below:
```
Comment (JamesHR3) [inline content/en/docs/corda-enterprise/4.4/tools-config-obfuscator.md:269]: I've noticed this sentence still makes reference to the hardware address, which only matters if the non-recommended defaults are used. I guess this should really say that when the node/service is run with the correct seed/passphrase provided, then the configuration is deobfuscated on the fly.

Comment (ivanterziev-r3) [reply content/en/docs/corda-enterprise/4.4/tools-config-obfuscator.md:269]: Updated, thanks!

**Comment by ivanterziev-r3** [comment]
@rnicoll please take a look as well

**Comment by ivanterziev-r3** [comment]
Thanks @JamesHR3 - I'm afraid you need to re-approve the PR as I re-requested that at some point, sorry.

File: content/en/docs/corda-enterprise/4.4/tools-config-obfuscator.md [hunk 2]
```diff
@@ -17,37 +17,43 @@ title: Configuration Obfuscator
...
+The Configuration Obfuscator makes node installation less vulnerable to someone trawling plain text files, searching for passwords and credentials of resources that they should not have access to in the first place.
+
+{{< warning >}}
+Although the Configuration Obfuscator does protect the node against trawling attacks, it does not ensure that password protection is completely secure. For improved security, always use the [latest released version](https://docs.corda.net/docs/corda-enterprise/tools-config-obfuscator.html) of the tool.
+{{< /warning >}}
 
 {{< note >}}
-The tool makes node installation less vulnerable to someone trawling plain text files searching for passwords and
-credentials of resources that they should not have access to in the first place.
+The Configuraton Obfuscator tool can only be used with Corda Enterprise 4.4 (and above) node and Corda Firewall configuration files. Configuration files obfuscated with older versions of Corda Enterprise can still be deobfuscated by Corda Enterprise 4.4 and above.
```
Comment (rnicoll) [inline content/en/docs/corda-enterprise/4.4/tools-config-obfuscator.md:29]:
```suggestion
The Configuraton Obfuscator tool can only be used with Corda Enterprise 4.4 (and above) nodes, Corda Firewall and CENM 1.3 (and above) configuration files. Corda configuration files obfuscated with older versions of Corda Enterprise can still be deobfuscated by Corda Enterprise 4.4 and above. CENM 1.1 and 1.2 obfuscated configuration files can be deobfuscated with CENM 1.3 and above, but should be re-obfuscated with the new tool.
```

Comment (rnicoll) [reply content/en/docs/corda-enterprise/4.4/tools-config-obfuscator.md:29]: After discussion, realised this content is for 4.5, not 4.4. Please ignore.

**Pull request merged**